docs/src/inscriptions/recursion.md

```diff
@@ -26,11 +26,70 @@ The recursive endpoints are:
 - `/r/blockhash`: latest block hash.
 - `/r/blockhash/<HEIGHT>`: block hash at given block height.
 - `/r/blocktime`: UNIX time stamp of latest block.
-- `/r/metadata/<INSCRIPTION_ID>`: returns a JSON string containing the hex-encoded CBOR metadata.
+- `/r/metadata/<INSCRIPTION_ID>`: JSON string containing the
+  hex-encoded CBOR metadata.
+- `/r/sat/<SAT_NUMBER>`: the first 100 inscription ids on a sat.
+- `/r/sat/<SAT_NUMBER>/<PAGE>`: the set of 100 inscription ids on `<PAGE>`.
+- `/r/sat/<SAT_NUMBER>/at/<INDEX>`: the inscription id at `<INDEX>` of all
+  inscriptions on a sat. `<INDEX>` may be a negative number to index from the
+  back.
 
-Responses from the above recursive endpoints are JSON. For backwards compatibility additional endpoints are supported, some of which return plain-text responses.
+Note: `<SAT_NUMBER>` only allows the actual number of a sat no other sat
+notations like degree, percentile or decimal. We may expand to allow those in
```

Comment:
Is this difficult to implement? The traditional `/sat` endpoint didn't care which notation used and I'm using that quite a bit. A point in the right direction and I could take a stab at it

Reply:
We talked about how that adds complexity for other people needing to implement all those notations in their explorer to make this recursive endpoint. To avoid other explorers potentially not supporting certain inscriptions we went with just the sat number for now.

Reply:
fair enough, easy to convert on the other end anyways will just do that for now 🫡

Reply:
Yes, it's always easy to add the other notations as well. This way explorers have more time to get everything in order

Reply:
You can see our implementation in `/src/sat.rs`, `/src/degree.rs`, `/src/decimal.rs`. We can always accept more notations later. It's a couple line change on our end. Do you want to implement it for your explorer/infrastructure?